yvalenta/Example-socket/simple-client.rb

```diff
@@ -0,0 +1,12 @@
+require 'socket'      # Sockets are in standard library
+
+hostname = '127.0.0.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.